lib/Runtime/Library/JavascriptObject.cpp

```diff
@@ -1096,6 +1096,92 @@ namespace Js
         return JavascriptOperators::GetOwnEnumerablePropertyNames(object, scriptContext);
     }
 
+    Var JavascriptObject::GetValuesOrEntries(RecyclableObject* object, bool valuesToReturn, ScriptContext* scriptContext)
+    {
+        Assert(object != nullptr);
+        Assert(scriptContext != nullptr);
+        JavascriptArray* valuesArray = scriptContext->GetLibrary()->CreateArray(0);
+
+        Var ownKeysVar = JavascriptOperators::GetOwnEnumerablePropertyNames(object, scriptContext);
```

Comment:
Assume we have a non-enumerable property when this call is made. Between here and the `IsEnumerable` call on line 1134, it looks like it's possible to make the property enumerable. Wouldn't it be excluded from the result?

I see that we have tests for the opposite direction (i.e., enumerable -> non-enumerable), but we're missing this case.